acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationWithAttributeConnectorIT.java

```diff
@@ -103,16 +95,16 @@
     @Autowired
     private PolicyHelper policyHelper;
 
-    private static final String IDENTIFIER = "part/03f95db1-4255-4265-a509-f7bca3e1fee4";
-    private static final String ASSET_URI_PATH_SEGMENT = '/' + IDENTIFIER;
+    private static final String TEST_PART_ID = "part/03f95db1-4255-4265-a509-f7bca3e1fee4";
+    private static final String ASSET_URI_PATH_SEGMENT = '/' + TEST_PART_ID;
 
     private Zone zone;
     private OAuth2RestTemplate acsAdminRestTemplate;
     private OAuth2RestTemplate assetRestTemplate;
     private boolean registerWithZac;
-    private URI resourceAttributeConnectorUrl;
+    private URI resourceAttributeConnectorUrl = URI.create(this.zoneHelper.getAcsBaseURL() + "/v1/resource/connector");
```

Comment:
This will throw a `NullPointerException` when the test is run since `this.zoneHelper` isn't initialized yet. We should move this initialization to the `@BeforeClass` method below.

Also the connector URI is `/v1/connector/resource` :) . It might just be better to use `AcsApiUriTemplates.V1 + AcsApiUriTemplates.RESOURCE_CONNECTOR_URL` to construct the URI path like before so that we only have to make a change in one place.